### PR TITLE
[MaximumLikelihoodMSM] convert sparse counts to dense based on flag sparse

### LINKED ISSUE #404: [msm.estimate_markov_model] parameter sparse=False gets overriden by method=auto of [msm/estimation/api/tmatrix]
Somehow connected with #403

### PR DESCRIPTION
**Pull request by marscher**
Fixes #404

### CODE REVIEW

File: pyemma/msm/estimators/maximum_likelihood_msm.py [hunk 1]
```diff
@@ -1093,19 +1102,6 @@ def _estimate(self, dtrajs):
             P = msmest.transition_matrix(self._C_active, reversible=self.reversible,
                                          mu=statdist_active,
                                          maxiter=self.maxiter, maxerr=self.maxerr)
-        else:
-            raise NotImplementedError(
-                'MSM estimation with connectivity=%s is currently not implemented.' % self.connectivity)
```
Comment (marscher) [inline pyemma/msm/estimators/maximum_likelihood_msm.py:1098]: this check is performed in the base class.

**Comment by gph82** [comment]
I would wait to see what @franknoe says, though

**Comment by franknoe** [comment]
I currently don't fully understand the relevance of this change, but the MSM classes are thoroughly tested, so if the tests run I'm not concerned.

**Comment by marscher** [comment]
We skip the Mac tests for now, because of https://www.traviscistatus.com/incidents/mc9x2wmpnvhg

**Pull request merged**